### PR TITLE
added initial scaffolding for running containership prometheus agents

### PR DESCRIPTION
**Pull request by nicktate**
@normanjoyner 

Need to activate `cadvisor`. Currently debugging why the downloaded executable is not able to run. Node_exporter works and is turned on.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+RUN n $NODE_VERSION
+RUN npm install
+
+WORKDIR $PROMETHEUS_AGENT_HOME/agents
+
+# CADVISOR
+ENV PROM_AGENT_CADVISOR=true
+ENV PROM_AGENT_CADVISOR_VERSION=v0.23.2
+ENV PROM_AGENT_CADVISOR_PORT=8500
+ENV PROM_AGENT_CADVISOR_CMD="sudo $PROMETHEUS_AGENT_HOME/agents/cadvisor/cadvisor"
```
Comment (normanjoyner) [inline Dockerfile]: is there a reason this needs sudo?

**Comment by normanjoyner** [comment]
README documentation for env vars

File: Dockerfile (hunk 1 above)
Comment (nicktate) [inline Dockerfile]: https://github.com/google/cadvisor/blob/master/docs/running.md#standalone

Comment (nicktate) [inline Dockerfile]: They use sudo in all their examples stating some services my require root access to monitor.

**Comment by nicktate** [comment]
@normanjoyner updated

**Pull request merged**